pkg/kuttlctl/cmd/assert.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+
+	harness "github.com/kudobuilder/kuttl/pkg/apis/testharness/v1beta1"
+	"github.com/kudobuilder/kuttl/pkg/test"
+)
+
+var (
+	assertExample = `  # Asserts against a $KUBECONFIG cluster the values defined in the assert file.
+  kubectl kuttl assert <path/to/assertfile.yaml>`
```

Comment:
I'd make this accept any number of files, so that one can simply do `kubectl kuttl assert .../*-assert.yaml` rather than build `for` loops in bash...

Reply:
haha... I should have just done that... there was a voice in my head wondering if you would appreciate it... I will listen to it next time (and now)